packages/realm/package.json

```diff
@@ -51,12 +51,12 @@
     "test": "wireit",
     "test:types": "tsc --project type-tests/tsconfig.json",
     "lint": "eslint --ext .js,.mjs,.ts .",
-    "build": "wireit",
+    "build": "echo 'Run bundle, build:node, build:ios or build:android or other scripts depending on these instead' && exit 1",
```

Comment:
This change is potentially controversial. I don't see a use for building for all platforms now that tests invoke the platform specific build commands transitively and on CI it's also split into separate runners.

Reply:
That's a good change.  I constantly find myself having to cancel that task.  The `ios` build script doesn't respond immediately to `ctrl-c`